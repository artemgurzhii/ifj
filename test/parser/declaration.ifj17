let a, b: int
let a = 5
let a: string = "foo"