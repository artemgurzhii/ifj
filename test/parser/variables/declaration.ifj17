dim foo as string

dim
bar as
integer

dim  baz  as  string
