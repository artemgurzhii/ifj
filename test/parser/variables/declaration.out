(dim
  (decl
    (id foo)
    : (id string)))

(dim
  (decl
    (id bar)
    : (id integer)))

(dim
  (decl
    (id baz)
    : (id string)))
